packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -965,6 +965,25 @@ - (void)invalidate {
                              }];
 }
 
+RCT_EXPORT_METHOD(unenrollMultiFactor
```

Comment:
iOS implementation is untested, but should be _**roughly**_ correct. If someone could give it a whirl that would be helpful